examples/with-sentry/server.js

```diff
@@ -0,0 +1,66 @@
+const next = require('next')
+const express = require('express')
+const cookieParser = require('cookie-parser')
+const Sentry = require('@sentry/node')
+const uuidv4 = require('uuid/v4')
+const port = parseInt(process.env.PORT, 10) || 3000
+const dev = process.env.NODE_ENV !== 'production'
+const app = next({ dev })
+const handle = app.getRequestHandler()
+
+require('./utils/sentry')
+
+app.prepare()
+  .then(() => {
+    const server = express()
+
+    // This attaches request information to sentry errors
+    server.use(Sentry.Handlers.requestHandler())
+
+    server.use(cookieParser())
+
+    server.use((req, res, next) => {
```

Comment:
I don't think this part is needed

Reply:
why? it's needed for tracking users for errors in sentry

Reply:
At the very least I don't think it should use an express middleware but use getInitialProps instead

Reply:
it won't work if next.js crashes before of while excuting getIntialProps

Reply:
Also I can't use getInitialProps  because session id for user won't be preserved at all cases (e.g. when someone does full refresh of webpage). It would require reimplementing cookies in local storage.